code/modules/mob/mob.dm

```diff
@@ -34,7 +34,6 @@
 			continue
 		var/datum/atom_hud/alternate_appearance/AA = v
 		AA.onNewMob(src)
-	set_nutrition(rand(NUTRITION_LEVEL_START_MIN, NUTRITION_LEVEL_START_MAX))
```

Comment:
doesn't happen for slimes anymore?

Reply:
good point thanks

Reply:
they automatically start with 700 nutrition already so this shouldnt do anything

Reply:
they used to but that got PR'd out since it gave babbies random nutrition too iirc

Reply:
no i literaly checked the line is still there